sdk/python/mypy.ini

```diff
@@ -2,8 +2,6 @@
 
 [mypy]
 
-namespace_packages = True
```

Comment:
This was put in because protobuf broke namespacing a while back, they've since backed that change out so we can revert this as well.